tests/components/accuweather/test_init.py

```diff
@@ -58,3 +61,23 @@ async def test_unload_entry(hass):
 
     assert entry.state == ENTRY_STATE_NOT_LOADED
     assert not hass.data.get(DOMAIN)
+
+
+async def test_update_interval(hass):
+    """Test correct update interval."""
+    entry = await init_integration(hass)
+
+    assert entry.state == ENTRY_STATE_LOADED
+    assert hass.data[DOMAIN][entry.entry_id][COORDINATOR].update_interval == timedelta(
```

Comment:
We shouldn't assert integration details. Move time forward instead and check if a mock api call was made or not.

Reply:
OK, will do.